activerecord/lib/active_record/errors.rb

```diff
@@ -334,4 +334,10 @@ class Deadlocked < TransactionRollbackError
   # +reverse_order+ to automatically reverse.
   class IrreversibleOrderError < ActiveRecordError
   end
+
+  # UnknownAttributeReference is raised when an unknown and potentially unsafe
+  # value is passed to a query method when allow_unsafe_raw_sql is set to
+  # disabled.
```

Comment:
In the absence of a better place to put it, we should probably include a paragraph here on how to fix the exception (i.e. `Arel.sql`). And also be clearer about what an "unknown and potentially unsafe" value is.

Once the exception becomes the default, people are going to hit it regularly for a long time to come (either by following old experience, or old tutorials etc), so it seems important to explain how to do what they want.